bin/setup-runtime

```diff
@@ -52,6 +52,8 @@ china)
 
 esac
 
+apk add --no-cache curl || true
```

Comment:
It's working on Alpine, but other distros will show a warning message.

Reply:
warning hided